relay-dynamic-config/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "relay-dynamic-config"
+authors = ["Sentry <oss@sentry.io>"]
+description = "Dynamic configuration passed down from sentry"
+homepage = "https://getsentry.github.io/relay/"
+repository = "https://github.com/getsentry/relay"
+version = "23.1.1"
+edition = "2021"
+license-file = "../LICENSE"
+publish = false
+
+[features]
+default = []
+processing = []
```

Comment:
Please double-check, it looks like this feature flag is unused.

Reply:
Checked, the string `processing` does not appear anywhere else in the crate.